Doc/library/runpy.rst

```diff
@@ -120,19 +120,19 @@ The :mod:`runpy` module provides two functions:
 
    The special global variables ``__name__``, ``__spec__``, ``__file__``,
    ``__cached__``, ``__loader__`` and ``__package__`` are set in the globals
-   dictionary before the module code is executed (Note that this is a
+   dictionary before the module code is executed. (Note that this is a
    minimal set of variables - other variables may be set implicitly as an
-   interpreter implementation detail).
+   interpreter implementation detail.)
 
    ``__name__`` is set to *run_name* if this optional argument is not
    :const:`None` and to ``'<run_path>'`` otherwise.
 
-   If the supplied path directly references a script file (whether as source
+   If ``file_path`` directly references a script file (whether as source
```

Comment:
Hi @maggyero  at line 27, there is an other reference to `supplied path`. 
Thank you

Reply:
Thanks, I had missed several of these!